backend/schema/errors.go

```diff
@@ -94,14 +94,15 @@ func Wrapf(pos Position, endColumn int, err error, format string, args ...any) *
 	return &e
 }
 
-func SortErrorsByPosition(merr []error) {
+func SortErrorsByPosition(merr []*Error) {
+	if len(merr) == 0 {
+		return
+	}
```

Comment:
Mostly just for my info, but is this check needed? Would just `sort.Slice` on an empty slice do the same thing?

Reply:
nope you're totally right - removed!